README.md

```diff
@@ -1,2 +1,18 @@
-# lc3-virtual-machine
-An implementation of a VM for the LC3 architecture
+# LC3 Virtual Machine
+
+##An implementation of a VM for the LC3 architecture
```

Comment:
If this is intended to be a header, leave a space between the `##` and the title